### ISSUE TITLE
High contrast glitch beta3 testing

### ISSUE THREAD

**Issue by sarah-beasley** [open-sdg/open-sdg#1845]
In the site footer link to the configuration page - http://uk-sdg-feature-branches.s3-website.eu-west-2.amazonaws.com/feature-2.1.0-beta3/config

On high contrast mode, the button to return to site repo has half black, half white text so not visible.

![image](https://user-images.githubusercontent.com/106670682/197748074-753dccfb-cc36-4e74-8c03-317d843a243f.png)

**Comment by brockfanning** [comment]
Just confirming, this is fixed in the latest 2.1.0-dev, so I'll go ahead and close.
